src/peer/component/mod.rs

```diff
@@ -242,7 +242,7 @@ impl State {
     }
 
     /// Returns all [`TrackId`]s of the peer.
-    pub fn get_tracks(&self) -> Vec<TrackId> {
+    pub fn get_tracks_ids(&self) -> Vec<TrackId> {
```

Comment:
@evdokimovs that was not a very good move, since the similar functions above are named without this `_ids` postfix. So it's better either to add the `_ids` postfix to them too, or to not add it here, for keeping the code style consistent.

I've removed the `_ids` postfix back here, since adding it to other function would be a more intrusive change. I propose you add it to all three functions as a separate PR.